stubs/qrcode/qrcode/main.pyi

```diff
@@ -1,13 +1,37 @@
-from _typeshed import Incomplete
-from typing import Generic, NamedTuple, TypeVar, overload
+from _typeshed import ConvertibleToInt, Incomplete
+from typing import Any, Generic, Literal, NamedTuple, TypeVar, overload
 from typing_extensions import TypeAlias
 
-from qrcode.image.base import BaseImage
+from .image.base import BaseImage
+from .util import QRData, _MaskPattern
 
 ModulesType: TypeAlias = list[list[bool | None]]
 precomputed_qr_blanks: dict[int, ModulesType]
 
-def make(data: Incomplete | None = None, **kwargs): ...
+_DefaultImage: TypeAlias = Any  # PilImage if Pillow is installed, PyPNGImage otherwise
+
+@overload
+def make(
+    data: QRData | bytes | str,
```

Comment:
This has a default of `None`, but at runtime this creates a QR code with the string `None`. This sounds not useful to me, but a potential source of bugs.